### PR TITLE
Show all refseq identifiers

### PR DESCRIPTION
**Pull request by moonso**
- Now all refseg transcripts will be shown in the transcripts overview and detailed table
- Stores all RefSeq identifiers for every transcript
- Display disease associated transcript from gene panels in original format on variant page

To get this to work one must run `scout update genes` after deploy

### CODE REVIEW

**Comment by moonso** [comment]
Hi @dnil , @northwestwitch this solves both problems described in #997 . Please have a look as soon as you have time.

**Comment by moonso** [comment]
Hi @dnil , @northwestwitch this solves both problems described in #997 . Please have a look as soon as you have time.

**Comment by northwestwitch** [comment]
I'll do the review now!

**Comment by northwestwitch** [comment]
I'll do the review now!

File: scout/load/transcript.py [hunk 1]
```diff
@@ -69,27 +69,33 @@ def load_transcripts(adapter, transcripts_lines=None, build='37', ensembl_genes=
             # If there are several mrna the one that is in 'primary_transcripts' is choosen
             # Else one is choosen at random
             # The same follows for the other categories where nc_rna has precedense over mrna_predicted
+            # We will store all refseq identifiers in a list as well
```
Comment (northwestwitch) [inline scout/load/transcript.py]: We will store all refseq identifiers in a "refseq_identifiers" list as well

File: scout/server/blueprints/variants/templates/variants/variant.html [hunk 2]
```diff
@@ -489,25 +486,29 @@
...
-                <div>{{ transcript.refseq_id or transcript.transcript_id}}</div>
-              </td>
-              <td title="{{ transcript.coding_sequence_name or '' }}">
-                {{ (transcript.coding_sequence_name or '')|truncate(20, True) }}
-                <span class="text-muted pull-right">
-                  {{ (transcript.protein_sequence_name or '')|url_decode }}
-                </span>
-              </td>
-            </tr>
+            {% for refseq_id in transcript.refseq_identifiers %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/variant.html]: This table looks a bit too busy. I'd put the summary panel on top of the transcript one by modifying like 53 in this way:
```
<div class="row">
    <div class="col-xs-12">{{ matching_variants() }}</div>
    <div class="col-xs-12 col-sm-6 col-md-4">{{ panel_basics() }}</div>
    <div class="col-xs-12 col-sm-8 col-md-8">
      <div class="col-xs-12 col-sm-12 col-md-12">{{ panel_summary() }}</div>
      <div class="col-xs-12 col-md-12">{{ transcript_overview() }}</div>
    </div>
 </div>
```

**Comment by northwestwitch** [comment]
I'll do the review now!

**Pull request merged**